HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # Version History
 
+## 0.1.4-preview / 2021-08-06
```

Comment:
if we are breaking things i might consider it .2

Reply:
I changed it to 0.2.0, and also added a note explicitly stating the function's old and new names in the history file